### DISCUSSION TITLE
Font and Color of your Templates

### DISCUSSION THREAD

**Discussion by mrdarcilite** [BryanJenksCommunity/FAQ#131] [Q&A] [answered]
Hello Bryan! 

THANK YOU SO MUCH for everything that you have contributed to the community. I have benefitted greatly from your Obsidian workflows.

However, I was hoping if there was a way we could change the font and color of your obsidian template (as I generally prefer something less strong). Is it possible!?

Thanks

**Comment by tallguyjenks** [comment, marked as answer]
of course you can, you can modify anything in the CSS as you have a copy of the file, so go nuts
